Python/Product/VSInterpreters/CPythonInterpreterFactoryProvider.cs

```diff
@@ -180,6 +181,13 @@ class CPythonInterpreterFactoryProvider : IPythonInterpreterFactoryProvider {
                     version = new Version(2, 7);
                 }
 
+                try {
+                    var langVer = version.ToLanguageVersion();
+                } catch (InvalidOperationException) {
+                    // Version is not currently supported
+                    return;
+                }
```

Comment:
This is needed to prevent a crash when you have Python 3.6 installed (which is currently not supported, but gets through most of the later checks).